test/utils/browser.js

```diff
@@ -30,32 +32,86 @@ exports.openMusic = function (options) {
   // Execute many async steps
   before(function startBrowser () {
     this.browser = wd.remote();
+    process.argv.forEach(function eachArgument (arg) {
+      // Uniquely identify a browserstack build based on timeout
+      if (arg === '40000') {
+        this.browser = wd.remote('hub.browserstack.com', 80);
+      }
+    }.bind(this));
```

Comment:
I considered how bad the "magic argument" would be and weighed it up against ease of testing on your local machine, if you feel that switching between local and remote by changing an environment variable isn't too cumbersome then I'll make this change :)